src/test/scala/com/tsunderebug/scolortesting/otf/types/SequentialMapGroupTest.scala

```diff
@@ -0,0 +1,42 @@
+package com.tsunderebug.scolor.otf.types
+
+import org.scalatest.{FlatSpec, OptionValues}
+import spire.math.UInt
+
+import com.tsunderebug.scolor.otf.{OTFByteAllocator, OpenTypeFont}
+
+class SequentialMapGroupTest() extends FlatSpec with OptionValues {
+
+  private def byteAllocator = new OTFByteAllocator(OpenTypeFont(Nil))
```

Comment:
Since the ByteAllocator has no effect on the length or sections computation for this class I just passed `Nil` for the tables for the `OpenTypeFont` class

Reply:
:ok_hand: